pages/linux/journalctl.md

```diff
@@ -16,7 +16,7 @@
 
 - Show all messages by a specific unit:
 
-`journalctl -u {{unit}}`
+`journalctl -u {{unit}} --since {{today}} --until {{YYYY-MM-DD HH:MM:SS}}`
```

Comment:
This can be a separate example by itself. Also please add the strings `yesterday|tomorrow` to `since`. And clarify that either of them can be used in since or until